src/app/api/util/stringPixels.ts

```diff
@@ -0,0 +1,13 @@
+export class StringPixels {
+    characterSize = 7;
+
+    /**
+     * Returns an estimate of the width that may have a text rendered based on
+     * a character constant
+     * @param {String} text The text to be measure.
+     *
+     */
+    public getTextWidth(text: string): number {
+        return this.characterSize * text.length;
```

Comment:
We need to return this the max-width, like 

```this.characterSize * text.length > 140 ? 140 : this.characterSize * text.length```

We can't set that in the css because in some place we're going to need to do ```width: 100%``` and the ```max-width``` in the css won't allow us to do that.